x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/findings_by_resource_table.test.tsx

```diff
@@ -27,9 +27,9 @@ const getFakeFindingsByResource = (): FindingsByResourcePage => {
   return {
     cluster_id: chance.guid(),
     resource_id: chance.guid(),
-    resource_name: resourceName,
-    resource_subtype: resourceSubtype,
-    cis_sections: cisSections,
+    'resource.name': resourceName,
+    'resource.sub_type': resourceSubtype,
+    'rule.section': cisSections,
```

Comment:
keys changed to match the keys (properties) on the actual document. 

introducing these intermediate keys on entry (this is how we passed data to view) would require us to transform them when filtering is applied, as these fields can't be filtered. so it's better to avoid it.

in addition, there's [relevant](https://github.com/orouz/kibana/commit/7db82f230959eb881efc9d6700fabc13a6401947#diff-20edd9aeca1692c60b676c9ddde1079d185fca50948e2f5baf41d3dd8889822dR16-R49) work done on going with this approach (using `nested.keys.strings`) when we move from consuming data from `_source` to `fields` 

we should probably discuss this further